src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml

```diff
@@ -68,6 +68,12 @@ mb:
             layerOrder: Layer ordering
             layerOrder.standard: standard
```

Comment:
- please fix: upper case to lower case
- . in the variable
            layerOrder.standard: standard
            layerOrder.reverse: QGIS